src/org/zaproxy/zap/extension/exportreport/model/AlertItem.java

```diff
@@ -0,0 +1,308 @@
+	{
+		return otherinfo;
+	}
+
+	@XmlElement(name = "OtherInfo")
+	public void setOtherInfo(String otherinfo)
+	{
+		this.otherinfo = otherinfo;
+	}
+	// ---------------------------------------------------------------
+
+	public String getReference()
+	{
+		return reference;
+	}
+
+	@XmlElement(name = "Reference")
+	public void setReference(String reference)
+	{
+		this.reference = reference;
+	}
+	// ---------------------------------------------------------------
+
+	public String getCWEID()
+	{
+		return cweid;
+	}
+
+	@XmlElement(name = "CWEID")
+	public void setCWEID(String cweid)
+	{
+		this.cweid = cweid;
+	}
+	// ---------------------------------------------------------------
+
+	public String getWASCID()
+	{
+		return wascid;
+	}
+
+	@XmlElement(name = "WASCID")
+	public void setWASCID(String wascid)
+	{
+		this.wascid = wascid;
+	}
+	// ---------------------------------------------------------------
+
+	public String getURI()
+	{
+		return uri;
+	}
+
+	@XmlElement(name = "URI")
+	public void setURI(String uri)
+	{
+		this.uri = uri;
+	}
+	// ---------------------------------------------------------------
+
+	public String getParam()
+	{
+		return param;
+	}
+
+	@XmlElement(name = "Param")
+	public void setParam(String param)
+	{
+		this.param = param;
+	}
+	
+	// ---------------------------------------------------------------
+
+	public String getAttack()
+	{
+		return attack;
+	}
+
+	@XmlElement(name = "Attack")
+	public void setAttack(String attack)
+	{
+		this.attack = attack;
+	}
+
+	// ---------------------------------------------------------------
+
+	public String getEvidence()
+	{
+		return evidence;
+	}
+
+	@XmlElement(name = "Evidence")
+	public void setEvidence(String evidence)
+	{
+		this.evidence = evidence;
+	}
+	
+	// ---------------------------------------------------------------
+
+	public String getRequestHeader()
+	{
+		return requestheader;
+	}
+
+	@XmlElement(name = "RequestHeader")
+	public void setRequestHeader(String requestheader)
+	{
+		this.requestheader = requestheader;
+	}
+	
+	// ---------------------------------------------------------------
+
+	public String getRequestBody()
+	{
+		return requestbody;
+	}
+
+	@XmlElement(name = "RequestBody")
+	public void setRequestBody(String requestbody)
+	{
+		this.requestbody = requestbody;
+	}
+	
+	// ---------------------------------------------------------------
+
+	public String getResponseHeader()
+	{
+		return responseheader;
+	}
+
+	@XmlElement(name = "ResponseHeader")
+	public void setResponseHeader(String responseheader)
+	{
+		this.responseheader = responseheader;
+	}
+	
+	// ---------------------------------------------------------------
+
+	public String getResponseBody()
+	{
+		return responsebody;
+	}
+
+	@XmlElement(name = "ResponseBody")
+	public void setResponseBody(String responsebody)
+	{
+		this.responsebody = responsebody;
+	}
+	
+	// ---------------------------------------------------------------
+	
+	public List<String> getArtists()
```

Comment:
I'll check this out in an hour or two and do anther commit, i might have accidentally reverted to an older version or forgot to change the name or remove an unused method.

Did you want me to try and squash so it's one commit?